<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenGUI</class>
 <widget class="QWidget" name="OpenGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/game.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="target">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>134</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/ten.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="red">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>149</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/red.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="blue">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>149</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/blue.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="tray">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>121</width>
     <height>82</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/tray.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="num_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/num_3.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="num_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/num_2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="sta">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/start.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="TimeL">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 48pt &quot;Adobe Garamond Pro Bold&quot;;
color: rgb(150, 150, 150);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="test">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/score.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/time.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="scoreL">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>440</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Garamond Pro Bold</family>
     <pointsize>48</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(148, 148, 148);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/mtaikoflower.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="cat">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>華康流隸體(P)</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 38);
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="cat_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>華康流隸體(P)</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 38);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bad">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>152</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/不可.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="good">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>148</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/良.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="num_1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="opengui.qrc">:/opengui/num_1.png</pixmap>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>target</zorder>
  <zorder>TimeL</zorder>
  <zorder>scoreL</zorder>
  <zorder>bad</zorder>
  <zorder>good</zorder>
  <zorder>blue</zorder>
  <zorder>red</zorder>
  <zorder>tray</zorder>
  <zorder>label_3</zorder>
  <zorder>cat_2</zorder>
  <zorder>cat</zorder>
  <zorder>test</zorder>
  <zorder>sta</zorder>
  <zorder>num_3</zorder>
  <zorder>num_2</zorder>
  <zorder>num_1</zorder>
 </widget>
 <resources>
  <include location="opengui.qrc"/>
 </resources>
 <connections/>
</ui>
